Makefile

```diff
@@ -1,6 +1,7 @@
 PHONY: .docker-build .build .run .mockgen
 PREFIX=/usr/local
-PUBLIC_JSON_SCHEMA_DIR=docs/schema/v0/
+JSON_SCHEMA_VERSION=v0.2.3
```

Comment:
can this be something for renovate based auto updates? [1] 🤔 

[1] - https://github.com/open-feature/flagd/blob/main/renovate.json

Reply:
I was thinking about that but I decided not to. My reasoning is that in many cases we will only want to serve a new schema with some thought and consideration, and probably at least an implementation in flagd (like for example @bacherfl change for weighing)

Reply:
Yes. I could also take it from an env var and just default it here. You only need to run it once and it creates the dirs (which correspond to the served URLs)

Reply:
This is fine, we just need to remember the change :)